sdk/spring/spring-cloud-azure-feature-management/src/test/java/com/azure/spring/cloud/feature/management/filters/recurrence/RecurrenceEvaluatorTest.java

```diff
@@ -0,0 +1,534 @@
+import com.azure.spring.cloud.feature.management.implementation.timewindow.recurrence.RecurrenceEvaluator;
+import com.azure.spring.cloud.feature.management.implementation.models.Recurrence;
+import com.azure.spring.cloud.feature.management.implementation.models.RecurrencePattern;
+import com.azure.spring.cloud.feature.management.implementation.models.RecurrenceRange;
+import org.junit.jupiter.api.Test;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RecurrenceEvaluatorTest {
+
+    @Test
+    public void dailyTrue() {
+        final ZonedDateTime now1 = ZonedDateTime.parse("2023-09-02T00:00:00+08:00");
+        final TimeWindowFilterSettings settings1 = new TimeWindowFilterSettings();
+        final RecurrencePattern pattern1 = new RecurrencePattern();
+        final RecurrenceRange range1 = new RecurrenceRange();
+        final Recurrence recurrence1 = new Recurrence();
+        pattern1.setType("Daily");
+        recurrence1.setRange(range1);
+        recurrence1.setPattern(pattern1);
+        settings1.setStart("2023-09-01T00:00:00+08:00");
+        settings1.setEnd("2023-09-01T00:00:01+08:00");
+        settings1.setRecurrence(recurrence1);
+        consumeEvaluationTestData(settings1, now1, true);
+    }
+
+    @Test
+    public void dailyMultiIntervalTrue() {
+        final ZonedDateTime now2 = ZonedDateTime.parse("2023-09-05T00:00:00+08:00");
+        final TimeWindowFilterSettings settings2 = new TimeWindowFilterSettings();
+        final RecurrencePattern pattern2 = new RecurrencePattern();
+        final RecurrenceRange range2 = new RecurrenceRange();
+        final Recurrence recurrence2 = new Recurrence();
+        pattern2.setType("Daily");
+        pattern2.setInterval(4);
+        recurrence2.setRange(range2);
+        recurrence2.setPattern(pattern2);
+        settings2.setStart("2023-09-01T00:00:00+08:00");
+        settings2.setEnd("2023-09-03T00:00:00+08:00");
+        settings2.setRecurrence(recurrence2);
+        consumeEvaluationTestData(settings2, now2, true);
+    }
+
+    @Test
+    public void dailyMultiIntervalTrue2() {
+        final ZonedDateTime now3 = ZonedDateTime.parse("2023-09-06T00:00:00+08:00");
+        final TimeWindowFilterSettings settings3 = new TimeWindowFilterSettings();
+        final RecurrencePattern pattern3 = new RecurrencePattern();
+        final RecurrenceRange range3 = new RecurrenceRange();
+        final Recurrence recurrence3 = new Recurrence();
+        pattern3.setType("Daily");
+        pattern3.setInterval(4);
+        recurrence3.setRange(range3);
+        recurrence3.setPattern(pattern3);
+        settings3.setStart("2023-09-01T00:00:00+08:00");
+        settings3.setEnd("2023-09-03T00:00:00+08:00");
+        settings3.setRecurrence(recurrence3);
+        consumeEvaluationTestData(settings3, now3, true);
+    }
+
+    @Test
+    public void dailyNumberedRangeFalse() {
```

Comment:
Can you add a testcase for the case that `Interval` is larger than 1 and `Range.Type` is `Numbered`